### ISSUE TITLE
Picture Target Binning Based on 4 Quadrants AND Number of Network Neurons

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#661]
Basing the binning scheme solely on the sum of pixel intensities in the 4 bins did not produce a skull, but it did produce a bit more variety in the images the PictureTargetTask was evolving. Having additional dimensions of variation in the MAP Elites archive could help. So, combine the 4 quadrant approach we already have with a dimension tracking the number of neurons (the way we did in the previous binning scheme).

Basically, you will make a new binning scheme that `extends` the 4 quadrant scheme. You will need to `@Override` nearly all of the methods, but you can also make use of some of the code in the previous version. You can call older methods with the same name using `super`. 

This new binning scheme will have 5 dimensions.

**Comment by schrum2** [comment]
Don't do any work on this for the moment. I have an idea for a way to make it easier to combine different binning schemes/dimensions together, and if it works, then any work spent on this issue will be wasted. However, if my combining idea works, then completing this issue should be trivially easy.

**Comment by schrum2** [comment]
I don't think this is worth implementing any more
